### PR TITLE
Replace download buttons on main BF downloads

### PR DESCRIPTION
**Pull request by hflynn**
Update BF download buttons for 5.1.4 release

### CODE REVIEW

**Comment by mtbc** [comment]
The new buttons may be seen at http://downloads.openmicroscopy.org/bio-formats/5.1.4-rc1/

**Comment by mtbc** [comment]
The API download icon says "txz" but actually links to a zip.

File: bf_downloads.html [hunk 1]
```diff
@@ -361,18 +361,18 @@
                                 <td><a
                                         href="http://github.com/openmicroscopy/bioformats"
                                         target="_blank"><img
-                                        src="images/bf_gh_repo.png"
+                                        src="images/github_repository.png"
                                         alt="GitHub Repository Link" /></a></td>
                             </tr>
                             <tr>
                                 <td><a href="http://git.openmicroscopy.org/"
```
Comment (mtbc) [inline bf_downloads.html]: Not this PR's issue, but I wonder if we want to link directly to http://git.openmicroscopy.org/?p=bioformats.git.

Comment (hflynn) [inline bf_downloads.html]: We do indeed, I'll fix that.

**Comment by hflynn** [comment]
Both issues should be fixed now thanks @mtbc 
@sbesson can we regenerate the rc1 downloads page?

**Comment by mtbc** [comment]
Done, and looking good. :+1:

**Comment by sbesson** [comment]
Thanks. Once Traivs is validated, all members of @openmicroscopy/release should be able to merge this PR.

**Comment by hflynn** [comment]
All green now so merging.

**Pull request merged**